backend/package.json

```diff
@@ -1,29 +1,29 @@
 {
- "name": "backend",
- "version": "1.0.0",
- "description": "",
- "main": "index.js",
- "scripts": {
-  "start": "set NODE_ENV=development && nodemon index.js",
-  "test": "set NODE_ENV=test && nodemon index.js"
- },
- "keywords": [],
- "author": "",
- "license": "ISC",
- "dependencies": {
-  "bcrypt": "^5.1.1",
-  "compression": "^1.7.4",
-  "cookie-parser": "^1.4.6",
-  "cors": "^2.8.5",
-  "dotenv": "^16.3.1",
-  "express": "^4.18.2",
-  "express-jwt": "^8.4.1",
-  "express-validator": "^7.0.1",
-  "helmet": "^7.0.0",
-  "joi": "^17.11.0",
-  "joi-password-complexity": "^5.2.0",
-  "jsonwebtoken": "^9.0.2",
-  "mongoose": "^7.5.3",
-  "nodemon": "^3.0.1"
- }
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "set NODE_ENV=development && nodemon index.js",
+    "test": "set NODE_ENV=test && nodemon index.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "bcrypt": "^5.1.1",
+    "compression": "^1.7.4",
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "express-jwt": "^8.4.1",
+    "express-validator": "^7.0.1",
+    "helmet": "^7.0.0",
+    "joi": "^17.11.0",
+    "joi-password-complexity": "^5.2.0",
```

Comment:
i don't this it is required you can remove and create your own code instead of this